aten/src/THCUNN/generic/AbsCriterion.cu

```diff
@@ -7,13 +7,12 @@ void THNN_(AbsCriterion_updateOutput)(
            THCTensor *input,
            THCTensor *target,
            THCTensor *output,
-           bool sizeAverage,
-           bool reduce)
+           int64_t reduction)
```

Comment:
here and everywhere else in decl: does using `Reduction` instead of `int64_t` as type not work?

Reply:
I tried to use scoped enum at first, it doesn't work for jit and Zach told me it probably wasn't worth the effort to make jit support scoped enum.

Reply:
okay. Ideally here should have some comments saying `reduction` is an Enum so it will be easier for our future porter of THNN. But there are way too many instances. So idk if I should ask you to make this change.

Reply:
I put a comment in Reduction.h